.travis.yml

```diff
@@ -18,12 +18,12 @@ env:
     - DESTINATION="OS=9.2,name=iPhone 6s"      CONFIGURATION="Debug"
     - DESTINATION="OS=9.2,name=iPhone 6s"      CONFIGURATION="ReleaseTest"
     - DESTINATION="OS=9.3,name=iPhone 6s Plus" CONFIGURATION="Debug"
-    - DESTINATION="OS=9.3,name=iPhone 6s Plus" CONFIGURATION="ReleaseTest"    
+    - DESTINATION="OS=9.3,name=iPhone 6s Plus" CONFIGURATION="ReleaseTest"
     - DESTINATION="OS=10.0,name=iPhone 6s"      CONFIGURATION="Debug"
     - DESTINATION="OS=10.0,name=iPhone 6s"      CONFIGURATION="ReleaseTest"
 before_install:
   - gem install cocoapods --pre
-  - pod repo update
+  - pod repo update > /dev/null
```

Comment:
The `pod repo update` command generates a lot of output, so much (> 4Mb) that Travis terminates the build because the log file becomes too long. See job https://travis-ci.org/ciscospark/spark-ios-sdk/builds/176747005 , which terminated because of this. Silencing stdout solved this issue (job https://travis-ci.org/ciscospark/spark-ios-sdk/builds/176947555 ).